widgets/search-box/__tests__/search-box-test.js

```diff
@@ -168,6 +168,59 @@ describe('search-box()', () => {
     });
   });
 
+  context('input', () => {
+    let input;
+    beforeEach(() => {
+      container = document.createElement('div');
+      input = createHTMLNodeFromString('<input />');
+      input.addEventListener = sinon.spy();
+    });
+
+    context('instant search', () => {
+      beforeEach(() => {
+        widget = searchBox({container, autofocus: 'auto'});
+        widget.getInput = sinon.stub().returns(input);
+      });
+
+      it('do not perform the search on keyup', () => {
```

Comment:
I feel like this test is also not correctly named. It simply tests that we bind an event listener to `input` and tests that event listener.